lib/traject/config/sdr_config.rb

```diff
@@ -77,7 +77,9 @@ def stanford_mods(method, *args, default: nil)
 
 def mods_xpath(xpath)
   lambda do |resource, accumulator, _context|
-    accumulator << resource.mods.xpath(xpath, mods: 'http://www.loc.gov/mods/v3')
+    # Convert the xpath result (a Nokogiri nodeset) to a plain array.
+    # This allows traject methods like first_only to work properly.
+    accumulator.concat(resource.mods.xpath(xpath, mods: 'http://www.loc.gov/mods/v3')).to_a
```

Comment:
```suggestion
    accumulator.concat(resource.mods.xpath(xpath, mods: 'http://www.loc.gov/mods/v3'). to_a)
```

Is the `#to_a` misplaced, or maybe it's unnecessary (because concat calls it implicitly)

Reply:
@cbeer thanks - you're right, .concat does the conversion. With this change everything works the same as expected.